flight/util/json_parsing.py

```diff
@@ -0,0 +1,65 @@
+"""Separates data from SUAS mission plan into data structures for use in specific files"""
+from typing import Dict, List
+import json
+
+
+def waypoint_parsing(filename: str) -> List[Dict[str, float]]:
+    """
+    Accepts name of JSON file and extracts waypoint data for SUAS mission
+
+    Parameters
+    ----------
+        filename: str
+            String of data file to open and access waypoint data
+
+    Returns
+    -------
+        List[Dict[str, float]]
+            List of dictionaries containing latitude, longitude and altitude of each waypoint in mission
+
+    Raises
+    ------
+        General
+            Designed to detect any error to prevent data corruption and always close the file being read
```

Comment:
This doesn't actually raise this.
Instead you're technically "catching" an exception, which is not something you need to mention in the docstring.